lm/srilm.py

```diff
@@ -94,6 +95,82 @@ def hash(cls, kwargs):
         return super().hash(kwargs)
 
 
+class DiscountNgramsJob(Job):
+    """
+    Create a file with the discounted ngrams with SRILM
+    """
+
+    def __init__(
+        self,
+        ngram_order: int,
+        counts: tk.Path,
+        vocab: tk.Path,
+        count_exe: tk.Path,
+        *,
+        extra_discount_args: Optional[List[str]] = None,
+        cpu_rqmt: int = 1,
+        mem_rqmt: int = 48,
+        time_rqmt: float = 24,
+        fs_rqmt: str = "100G",
```

Comment:
Why is this different from the other requirements (in not being an int)? Is this some practical benefit because of how it is passed within the job?

Reply:
this requirement is not implemented per se in sisyphus, rather we use the fallback option "add any arbitrary string to the SGE call".. (cf. line 139)

Speaking of which.. @christophmluscher should `qsub_args` not rather be `sbatch_args` now for slurm?

Reply:
My suggestion would be to add both qsub_args and sbatch_args, the engine anyway only picks the needed entry.

Reply:
>Why is this different from the other requirements (in not being an int)? Is this some practical benefit because of how it is passed within the job?

I am staying consistent with the other jobs in this regard. I dont remember the design decision for `str`